test/providers/dynect_test.rb

```diff
@@ -40,6 +40,23 @@ def test_build_aaaa_from_api
     assert_equal 60, record.ttl
   end
 
+  def test_build_alias_from_api
+    record = @dyn.send(:build_from_api, {
+      "zone" => 'dns-test.shopify.io',
+      "ttl" => 60,
+      "fqdn" => "alias.dns-test.shopify.io",
+      "record_type" => "ALIAS",
+      "rdata" => {
+        "cname" => "dns-test.shopify.io."
```

Comment:
Right, but the [API docs specify](https://help.dyn.com/create-alias-records-api/) the attribute `alias`. This code will break right now if it tries to interact with DynECT's API.